projects/mosquitto/project.yaml

```diff
@@ -0,0 +1,13 @@
+homepage: "https://mosquitto.org/"
+language: c
+primary_contact: "roger@atchoo.org"
+vendor_ccs:
+ - "security@eclipse.org"
+auto_ccs:
+ - "rogerlight@gmail.com"
+sanitizers:
+ - address
+ - memory:
+    experimental: True
```

Comment:
Did you copy this from somewhere? Otherwise I wouldn't mark msan as experimental.

Reply:
Yes, from the documentation:

https://google.github.io/oss-fuzz/getting-started/new-project-guide/#sanitizers